packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -91,22 +108,34 @@ const FolderTreeItem = ({
       <li
         className={itemClassNames}
         role="treeitem"
+        aria-expanded={isFolder ? isExpanded : null}
         aria-selected={isSelected}
+        onClick={onToggle}
+        onKeyDown={onToggle}
       >
         <input
           type="radio"
           checked={isSelected}
-          onClick={onClick}
+          onChange={onClick}
+          aria-hidden // Hiding the radio button from assistive technology since they cannot be grouped correctly
+          tabIndex={-1} // Prevent tabbing to the button since it should not be read or acknowledged by assistive technology
```

Comment:
This will allow the control to get focus when you click though.  Is that what you want?

Reply:
We want the radio button to be clickable (per the previous story's AC) but we need to hide it from screenreaders so that they aren't reading radio buttons out of the tree hierarchy.